h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -560,7 +561,7 @@ def _metrics_class(model_json):
             model_class = H2OAnomalyDetectionModel
         elif model_type == "CoxPH":
             metrics_class = H2OModelMetricsRegressionCoxPH
-            model_class = H2OCoxPHModel
+            model_class = H2OCoxPHModel if model_json["algo"] != "generic" else H2OCoxPHMojoModel
```

Comment:
do we keep the "generic" solution for Mojo models for a follow up ticket? or was it an issue mainly for CoxPH?

Meaning… couldn't we override `model_class` globally after this sequence of if-else:
```python
if model_json['algo'] == 'generic':
    model_class = ModelBase   # or define a GenericModel class
```

Reply:
@sebhrusen we cannot do it globally because the more common models like GBM behave okay - that would be a regression in behavior

I do not believe CoxPH is the only model to suffer from an issue but we need to test it first

In this fix, I wanted to be just focused on the issue that was reported.